src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int DEF_CAPACITY = 1 << 4;
```

Comment:
```suggestion
    private static final int DEFAULT_CAPACITY = 1 << 4;
```
Do not use names abbreviations. Also, it is better to use value instead arithmetic operations when you initialize a constant field

Reply:
done